das-dashboard/backend/services/container_services.py

```
@@ -39,56 +58,66 @@ def manage_container(
+
                 if not self._is_remote(host):
                     has_local_command = True
 
                 commands_to_run[cmd_name] = self.build_das_cli_command(
                     host=host, service=service, action=action.value
                 )
-            except DASServiceInstantiationError:
-                continue
 
+            except DASServiceInstantiationError as exc:
+                raise ValueError(f"Service '{cmd_name}' is not configured.") from exc
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)